evolve_keyboard_layout_wasm/www/app.js

```diff
@@ -338,15 +341,22 @@ Vue.component('evaluator-app', {
 
         async startOptimization() {
             if (this.optMode === "simulated_annealing") {
-                this.saOptimization()
+                await this.saOptimization()
             } else if (this.optMode === "genevo") {
-                this.genevoOtimization()
+                await this.genevoOtimization()
```

Comment:
I forgot to delete this. If I understand correctly, it shouldn't do anything, as we're not trying to receive&unpack any `Promise`s at this point.